avalon/nuke/lib.py

```diff
@@ -24,6 +32,25 @@ def maintained_selection():
             [n['selected'].setValue(True) for n in previous_selection]
 
 
+def reset_selection():
+    """Deselect all selected nodes
+    """
+    for node in nuke.selectedNodes():
+        node["selected"] = False
+
+
+def select_nodes(nodes):
```

Comment:
Maybe `select_input_nodes` is more descriptive?

Reply:
I think adding `input` will give an impression that was used to select input (upstream) nodes of given nodes.

Reply:
Ahh, I only read the doc string. Name is good then.